### PR TITLE
docs: started work on the contributing.md guide

### PR DESCRIPTION
**Pull request by dpkreativ**
Here I created a general layout of how the documentation should look

@davydocsurg will be doing more work on it, so I'm adding my changes to the main repo so he can pull from there

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+- [Husky](https://typicode.github.io/husky/) - To manage githooks
+- [ESLint](https://eslint.org/) - To check for and prevent human errors in the codebase
+- [Prettier](https://prettier.io/) - To maintain proper formatting in the codebase
+- [Git](https://git-scm.com/) - To store updates to the codebase in commits. Here are a few basic tips to keep in mind
+- [GitHub](https://github.com/) - For collaboration. [Here's a crash course](https://www.youtube.com/watch?v=8Dd7KRpKeaE) that covers the basics
+- You are free to use any code editor you want; however, we strongly recommend using [Visual Studio Code](https://code.visualstudio.com/) for a good developer experience
+
+## Getting Started
+
+<!-- Add an introduction here -->
```
Comment (CeoFred) [inline CONTRIBUTING.md:40]: Where is the introduction?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+This guide will cover the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR.
+
+Use the table of contents icon <img src="public/assets/table-of-contents.png" width="24" height="24" /> on the top left corner of this document to get to a specific section quickly.
+
+# Ground Rules
+
+Read our [Code of Conduct](CODE_OF_CONDUCT.md) to understand how you can help keep our community approachable and respectable.
+
+### A quick summary
```
Comment (CeoFred) [inline CONTRIBUTING.md:15]: Is this necessary? if it is, then include it.

**Review by CeoFred — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
+# Contributing to Quizard (Frontend)
+
+<!-- Add a thank you message for new/existing contributors -->
```
[the remark this answers is not in the document]
Comment (dpkreativ) [reply CONTRIBUTING.md:3]: Yes. David is gonna work on that with his fork

File: CONTRIBUTING.md (hunk 2 above)
Comment (dpkreativ) [reply CONTRIBUTING.md:15]: Yeah it is. David will work on it as well

Basically everything in the comments is what he's focusing on

**Pull request merged**